packages/mantine/src/components/table/Table.tsx

```diff
@@ -175,6 +175,10 @@ export const Table: TableType = <T,>({
         onMount?.({...state, ...form.values});
     }, []);
 
+    const outSideClickRef = useClickOutside(() => {
+        table.resetRowSelection(true);
+    });
```

Comment:
Does it behave correctly when you have a predicate or date picker in the table?

(I'll be happy when we'll have the demo 👀 )

Reply:
I can try it, I still have my setup in a stash 👍 
i'll make you a video 📦

Reply:
🥲 Good call, I need to fix this 👍 

https://user-images.githubusercontent.com/63734941/200009231-ea122212-a442-4e84-a876-12abe5195f94.mp4

Reply:
Clicking in the table header is does not remove the selected row now

https://user-images.githubusercontent.com/63734941/200016827-e8197ab4-0742-43c8-b9f9-437832ed1872.mp4